docs/packages/pkg/Boost.rst

```diff
@@ -42,7 +43,13 @@ List of components (other libraries are header-only):
 - ``iostreams``
 - ``locale``
 - ``log``
-- ``math``
+- ``math`` - Requires Special names for ``find_package`` and ``target_link_libraries``, choose one of:
```

Comment:
This is a list of components that can be used in `hunter_add_package(Boost COMPONENTS ...)` so it doesn't belong here. For the `find_package(Boost CONFIG REQUIRED ...)` you can add code snippet similar to Qt:
* https://docs.hunter.sh/en/latest/packages/pkg/Qt.html#usage

Reply:
Shall make a new section for just `math` then, was trying not to expand it that far as it increases verbosity, but yeah it is probably required, shall do.